sdk/program/src/nonce.rs

```diff
@@ -0,0 +1,7 @@
+pub use solana_nonce::{state::State, NONCED_TX_MARKER_IX_INDEX};
+pub mod state {
+    pub use solana_nonce::{
+        state::{Data, DurableNonce, State},
```

Comment:
Oh never mind, I had it wrong -- `current` isn't a public module, so there's no need to re-export it. Thanks!